Dockerfile

```diff
@@ -6,7 +6,7 @@ WORKDIR /usr/src/sriov-network-metrics-exporter
 RUN make clean && make build
 
 FROM docker.io/alpine:3.16
-COPY --from=builder /usr/src/sriov-network-metrics-exporter/bin/* /usr/bin/
+COPY --from=builder /usr/src/sriov-network-metrics-exporter/build/* /usr/bin/
```

Comment:
can you please apply this change to Dockerfile.rhel also?

https://github.com/openshift/sriov-network-metrics-exporter/blob/master/Dockerfile.rhel#L12

Reply:
you are right!

done :)